### PR TITLE
Муравьев - Лабораторная работа #2

### PR DESCRIPTION
**Pull request by DenMur725**

### CODE REVIEW

**Comment by DenMur725** [comment]
@owerbat @VadimKutovoi

File: modules/big-int/include/big_int.h [hunk 1]
```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Muravev Denis
+
+#ifndef MODULES_BIG_INT_INCLUDE_BIG_INT_H_
+#define MODULES_BIG_INT_INCLUDE_BIG_INT_H_
+
+#include <string>
+#include <cstdint>
+#define BIGINT_MAX 1000000000
```
Comment (alexbatashev) [inline modules/big-int/include/big_int.h]: https://en.cppreference.com/w/cpp/types/numeric_limits ?

File: modules/big-int/include/big_int.h [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+#define MYINT64 int_fast64_t
```
Comment (alexbatashev) [inline modules/big-int/include/big_int.h]: using

File: modules/big-int/include/big_int.h [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+#define MYUINT64 uint_fast64_t
+
+class big_int {
```
Comment (alexbatashev) [inline modules/big-int/include/big_int.h]: Google C++ Code Style
https://google.github.io/styleguide/cppguide.html#Type_Names

**Comment by DenMur725** [comment]
@alexbatashev Fix define and rename class

**Pull request merged**